README.md

```diff
@@ -120,6 +121,19 @@ See the documentation on the Waybill CRD
 [spec](https://godoc.org/github.com/utilitywarehouse/kube-applier/apis/kubeapplier/v1alpha1#WaybillSpec)
 for more details.
 
+#### Delegate Service Account
+
+To avoid leaking access from kube-applier to client namespaces, the concept of a
+delegate Service Account is introduced. When applying a Waybill, kube-applier
```

Comment:
I would write it as "Delegate ServiceAccount".

Delegate capitalised, and ServiceAccount together.